pom.xml

```diff
@@ -171,6 +171,7 @@
         <dependency>
             <groupId>ch.qos.reload4j</groupId>
             <artifactId>reload4j</artifactId>
+            <version>1.2.19</version>
```

Comment:
Could we pin this version in properties section similar to how we have done for slf4j - https://github.com/confluentinc/kafka-connect-jdbc/blob/10.0.x/pom.xml#L64? 

Also is this dependency version not present in commons version?

Reply:
Thanks @aniketshrimal 
I have pinned the version as a property.

Yes, the common-parent (common-parent-6.0.0) does not define the reload4j.version property. Hence, we need to add the version separately.
The property was defined for common-parent of 5.4.x and 5.5.x branches. 
`<reload4j.version>1.2.19</reload4j.version>`